includes/class-freemius.php

```diff
@@ -894,8 +933,9 @@ private function _find_caller_plugin_file( $is_init = false ) {
 				if ( isset( $this->_storage->plugin_main_file ) &&
 				     isset( $this->_storage->plugin_main_file->prev_path )
 				) {
-					if ( file_exists( $this->_storage->plugin_main_file->prev_path ) ) {
-						return $this->_storage->plugin_main_file->prev_path;
+                    $absolute_path = $this->get_absolute_path( $this->_storage->plugin_main_file->prev_path );

```

Comment:
@fajardoleo I noticed that you've forgot to migrate `$this->_storage->plugin_main_file->prev_path` to the relative path structure (in the migration logic, not here).